documentation/source/simulator_support.rst

```diff
@@ -342,6 +342,7 @@ GHDL
 
     GHDL support in cocotb is experimental.
     Some features of cocotb may not work correctly or at all.
+    At least GHDL 2.0 is required.
```

Comment:
@ktbarrett wrote in https://github.com/cocotb/cocotb/pull/2924#discussion_r877099883:

> Earlier versions of GHDL do work, it was just that we added exemptions to get GHDL to pass the regression sometime before 2.0 released. So this sentence is not really correct.


GHDL 1.0 does not pass our test suite  as of today (I tried). And passing the test suite is the cocotb definition of "works with cocotb".

My goal here was to give our users guidance and let them know what we know. Of course, there might be use cases where GHDL 1.0 is sufficient. That's something users can then figure out if they really want/have to.

> I'd rather not make any promises about which versions of simulators re supported outside of "the latest release".

I'm not trying to make a promise (i.e. a positive support statement), but rather a negative one: versions below 2.0 are not working.